Watch Extension/FeedingView.swift

```diff
@@ -16,15 +18,43 @@ struct FeedingView: View {
     private func actionSheetButton(_ fip: Feeding) -> Alert.Button {
         if fip.isPaused {
             return .default(Text("Resume")) {
-                self.store.send(.feeding(.resume(fip)))
+                self.communicate(type: fip.type, side: fip.side, action: .resume)
             }
         } else {
             return .default(Text("Pause")) {
-                self.store.send(.feeding(.pause(fip)))
+                self.communicate(type: fip.type, side: fip.side, action: .pause)
             }
         }
     }
 
+    // TODO: consider making this a global function or in the session coordinator or ?
+    // This is the second time of copy and paste
+    func communicate(type: FeedingType, side: FeedingSide, action: Common.FeedingAction) {
+        let info = WatchCommunication(type: type, side: side, action: action)
+
+        let jsonEncoder = JSONEncoder()
+        guard let d = try? jsonEncoder.encode(info), WCSession.default.isReachable else {
+            // TODO: show communication error
+            return
+        }
+
+        WCSession.default.sendMessageData(d, replyHandler: { _ in
+            switch action {
+            case .start: break
+            case .stop:
+                self.store.send(.feeding(.stop(type: type)))
+            case .pause:
+                self.store.send(.feeding(.pause(type: type)))
+            case .resume:
+                self.store.send(.feeding(.resume(type: type)))
+            }
+        }) { error in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c7106bd91c271d949fbb3c9cc499512408cb0c64/Watch Extension/FeedingView.swift#L51"></span>Trailing closure syntax should not be used when passing more than one closure argument.<br />
<code>multiple_closures_with_trailing_closure</code> <code>FeedingView.swift:51</code></td>
    </tr>
  </tbody>
</table>